packages/volto/src/config/ControlPanels.js

```diff
@@ -90,6 +90,14 @@ export const filterControlPanelsSchema = (controlpanel) => {
       'picture_variants',
       'image_captioning',
     ],
+    navigation: [
+      'generate_tabs',
+      'navigation_depth',
+      'nonfolderish_tabs',
```

Comment:
This one is used here: https://github.com/plone/plone.restapi/blob/main/src/plone/restapi/services/navigation/get.py#L116

Reply:
I removed it.

Reply:
@davisagli even though `nonfolderish_tabs` is checked in restapi, its help text says:

```By default, any content item in the root of the portal will appear as a tab. If you turn this option off, only folders will be shown. This only has an effect if 'automatically generate tabs' is enabled.```

First, the term "tab" doesn't make sense in Volto. Second, the `automatically generate tabs` field has been removed. So, I think we should remove the nonfolderish_tabs field. Do you agree?

Reply:
@wesleybl I see, so the help text doesn't match how it is actually used in plone.restapi. Yeah, I think we should probably remove it from the control panel.